### PR TITLE
#163783282  Update Readme

### PR DESCRIPTION
**Pull request by MaggieChege**
**What does this PR do?**

 - Update the Readme

**Description of Task to be completed?**

-  Update the Readme File

#### Relevant Pivotal Tracker Stories

[#163783282](https://www.pivotaltracker.com/story/show/163783282)

### CODE REVIEW

**Review by andrewhingah — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+cd into ```ah-django-unchained-frontend```
+
+Install all dependencies:
+
+```npm install ```
+
+Start the project
+
+``` npm start ```
```
Comment (cosmas28) [inline README.md]: - Goodjob @MaggieChege. On top of that, I would urge you to add another section where you are going list all the stacks used in this project.

- please add section for testing too.

**Review by cosmas28 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MaggieChege) [reply README.md]: Okay adding that.

**Review by kwameasiago — APPROVED** [review]

**Review by Georgeygigz — APPROVED** [review]
Looks good to me

**Review by cosmas28 — CHANGES_REQUESTED** [review]
Under the `Stacks used in the Project` section I don't see testing stack, styling stack. Kindly add them.

**Review by cosmas28 — APPROVED** [review]

**Pull request merged**